### DISCUSSION TITLE
Should I be expecting stable-cascade to work in the dev branch right now? feature_extractor?

### DISCUSSION THREAD

**Discussion by tau0-deltav** [vladmandic/sdnext#2921] [Q&A]
I tried a few days ago and, same as today, I'm getting an error about the feature_extractor. on a pristine arch linux install with a 3090

```
05:12:47-296174 ERROR    Diffusers Failed loading model: models/Diffusers/models--stabilityai--stable-cascade/snapshots/ed84090b8756424c0b107fd277538e360c127c51 StableCascadeCombinedPipeline.__init__() got an unexpected keyword argument 'feature_extractor'
```

not even google recognises unexpected `feature_extractor`'s. not even deleting all references of it from the model's files cause any error other than this one.
I don't really wanna write up a full issue report with the absolutely forensic log (every typo and bash whoopsie!) of myself reinstalling SD.Next and ultimately downloading about 40 gigabytes of data over LTE... **if it's not supposed to be working**.

But I did follow the [wiki article to the letter](../wiki/Stable-Cascade)/clone different directory/a new venv/*[I omit a long issue report-like list of checks]*/pray to Bostrom/paperclip accident/torment nexus- 

point is I'm feeling frustrated and stupid so I'll ask here before I write this up properly.

(if anyone replies "oh didn't you see the discord announcement" I'm selling this GPU, flat lease and my identity to go live in a tree or something)

**Comment by vladmandic** [comment]
that's why we havent announced it - upstream changes are still happening daily and i cannot chase it constantly. as soon as upstream changes are stable and merged into diffusers, official support will be announced.

in the meantime, i've just updated sdnext code so it works fine with latest version of upstream branch.
